deploy/examples/create-external-cluster-resources.py

```diff
@@ -1190,6 +1196,12 @@ def create_rgw_admin_ops_user(self):
             self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME,
             "--caps",
             "info=read",
+            "--rgw-realm",
```

Comment:
We are just updating the caps,
Still not sure if we need these flags 
@thotz please confirm